pls_core/src/inflections/mod.rs

```diff
@@ -348,6 +348,18 @@ pub fn get_abbreviations_for_locale(
         abbrev_map.insert(i[0].clone(), i[2].clone());
         abbrev_map.insert(i[1].clone(), i[2].clone());
     }
+    let list_of_required_keys = [
```

Comment:
i am not really sure where the disconnect is.

if we replicate it here, we still don't solve the problem right? 

idea of this fallback is make things fool proof forever i.e. due to these getting added / changed in the templates, PLS will **never** crash, not for that reason at least.

connascence in https://www.youtube.com/watch?v=NLT7Qcn_PmI

not sure if we are clinging to the idea of a hashtable - if we are let's not be

Reply:
> idea of this fallback is make things fool proof forever i.e. due to these getting added / changed in the templates, PLS will never crash, not for that reason at least.
ah, I misunderstood this. So the appropriate thing to happen is to crash but crash gracefully(with explanations).